test/steps/options/1_given.js

```diff
@@ -72,6 +77,12 @@ export function anOptionsObjectWithVotesSetTo() {
 	this.test.ctx.options = { votes };
 }
 
+export function anOptionsObjectWithVotesSetToAndSignatureSetTo() {
+	const [votes, signature] = getQuotedStrings(this.test.parent.title);
+	const signatureBoolean = signature === 'true';
```

Comment:
This departs from our standards somewhat, of using quotes for strings and no quotes for booleans. It also makes it difficult for us to reuse this in a situation where we want to set `signature` to a string.